README.md

```diff
@@ -1,10 +1,12 @@
-
 # Coffee Tasks
 
 A [Hacktoberfest](https://hacktoberfest.com/) Present 🎁 to React Beginners
 
 ![image](https://user-images.githubusercontent.com/68426990/192261187-84c1029b-686c-4549-a41d-9f23394a98a9.png)
 
+# Created for **HACKTOBERFEST**! Come learn front-end development with us throughout the month of October!
+
+# **Description of the project:**
```

Comment:
This should have smaller font size that project title

Reply:
Do you want me to remove it altogether, or just lower the size then?

Reply:
You can remove `#Created for **HACK ...`
And lower the font size of `#** Description of the project`